tests/common/devices.py

```diff
@@ -323,7 +323,7 @@ def get_networking_uptime(self):
             return self.get_now_time() - datetime.strptime(start_time["ExecMainStartTimestamp"],
                                                            "%a %Y-%m-%d %H:%M:%S UTC")
         except Exception as e:
-            self.logger.error("Exception raised while getting networking restart time: %s" % repr(e))
```

Comment:
Doesn't need to be addressed in this PR, but it would probably be helpful to have a logger explicitly defined so we can quickly identify where these messages are coming from.